Plugins: Add missing docs for plugin catalog (#34679)
grafana/grafana: docs/sources/plugins/catalog.md

# Plugin catalog

> **Note:** Only available in Grafana v8.0+.

Plugin catalog allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the catalog.

![Plugin catalog install flow](/img/docs/plugins/plugins-catalog-user-install-8-0.gif)

To use the Plugin catalog, you first need to enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}).

> Before following the steps below, make sure you are logged in as a Grafana administrator.

Reviewer: Why is browse and uninstall before install instructions? Also, uninstall section is listed twice.

> Before following the steps below, make sure you are logged in as a Grafana administrator.

Author: Not sure what you mean as it's (in order) Browse, Install, Update and Uninstall?

> Before following the steps below, make sure you are logged in as a Grafana administrator.

Reviewer 2: @wbrowne, the required permissions are incorrect. You need to be an Organization Admin, not a Grafana Admin, in order to access the Configuration menu and the Plugin catalog.

> Before following the steps below, make sure you are logged in as a Grafana administrator.

Author: Great catch @oddlittlebird. Bug fixed https://github.com/grafana/grafana/pull/34773

## Browse plugins

1. In Grafana, navigate to **Configuration > Plugins**.

> 1. Click the **Install &amp; manage plugins** button.

Author: Let me see if that instance has the correct configuration and that your user is setup as an Admin

> 1. Click the **Install &amp; manage plugins** button.

Reviewer 2: Jita, are you able to see this? 
![image](https://user-images.githubusercontent.com/52059945/119723075-e891ce80-be21-11eb-929c-5b0ebefdfe38.png)

# Plugin catalog

The Plugin catalog, new in Grafana 8.0, allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the catalog.

<div class="medium-6 columns">
  <video width="800" height="500" controls>
    <source src="/assets/videos/plugins-catalog-install-8-0.mp4" type="video/mp4">
    Your browser does not support the video tag.
  </video>
</div>

Before you can use the Plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.
Before following the steps below, make sure you are logged in as a Grafana administrator.

## Browse plugins

1. In Grafana, navigate to **Configuration > Plugins**.
1. Click the **Install &amp; manage plugins** button.

> 1. You can now browse and search for plugins.

Reviewer 2: This is not a step.

> 1. You can also browse existing plugins by navigating to the **Library** tab.

Reviewer 2: This is not a step.

![Plugin catalog browse](/img/docs/plugins/plugins-catalog-browse-8-0.png)

## Install a plugin

1. In Grafana, navigate to **Configuration > Plugins**.

> 1. Click the **Install &amp; manage plugins** button.

Reviewer 2: Do not use "button." Just "Click **Install & manage plugins"**."

Though really, you could just rename the button **Manage plugins** and be fine.

1. Browse and find a plugin.
1. Click on the plugin logo.
1. Click the **Install** button. 
A confirmation message opens notifying that the installation was successful.

![Plugin catalog install](/img/docs/plugins/plugins-catalog-install-8-0.png)

## Update a plugin

1. In Grafana, navigate to **Configuration > Plugins**.
1. Click the **Install &amp; manage plugins** button.
1. Navigate to the **Library** tab.
1. Click on the plugin logo.

> 1. Click the **Update** button.

Reviewer 2: ```suggestion
1. Click **Update**.
```

A confirmation message opens notifying that the installation was successful.

![Plugin catalog update](/img/docs/plugins/plugins-catalog-update-8-0.png)

## Uninstall a plugin

1. In Grafana, navigate to **Configuration > Plugins**.
1. Click the **Install &amp; manage plugins** button.
1. Navigate to the **Library** tab.
1. Click on the plugin logo.

> 1. Click the **Uninstall** button.

Reviewer 2: ```suggestion
1. Click **Uninstall**.
```

# Plugin catalog

> The Plugin catalog, new in Grafana 8.0, allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the catalog.

Reviewer 2: ```suggestion
The Plugin catalog allows users to browse and manage plugins from within Grafana. Only Organization Admins who are also Grafana Server Admins can access and use the catalog.
```

Reviewer: Good point - we should verify and probably add this information as a Note.

<div class="medium-6 columns">
  <video width="800" height="500" controls>
    <source src="/assets/videos/plugins-catalog-install-8-0.mp4" type="video/mp4">
    Your browser does not support the video tag.
  </video>
</div>

> Before you can use the Plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.

Reviewer 2: @achatterjee-grafana, what are your thoughts on capitalization here? I would err on the side of not capitalizing it in most cases. Here, it could be read as a generic plugin catalog.

Reviewer: Agree, we should keep this as "Catalog".

Reviewer 2: Maybe "plugin catalog" for generic mentions and "Catalog" when you're talking about the specific screen? Not sure we want to necessarily capitalize every use of catalog, either. That would get annoying. I'm trying to come up with a coherent rule we could add to the style guide and drawing a blank.

Reviewer: Thinking over, let us stick to "plugin catalog".

> Before you can use the Plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.

Reviewer: ```suggestion
Before you can use the plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.
```

Before following the steps below, make sure you are logged in as a Grafana administrator.

> ## Browse plugins

Reviewer: ```suggestion
## Browse plugins

To browse available plugins:
```
